docs/build/dapp/01-introduction.md

```diff
@@ -0,0 +1,46 @@
+---
+id: introduction
+title: Howto create a DApp
+sidebar_label: Introduction
+---
+
+
+This documentation aims to guide beginner and intermediate developers in integrating with **Archethic Wallet** (**aeWallet**). It explains various communication methods and includes code snippets in **Dart (Flutter)** and **TypeScript**.
+
+
```

Comment:
We should also precise that the code examples for the web are in Javascript (instead of Typescript) and that our library handles Typescript perfectly.

Reply:
Explained in every JS/TS tab.